src/test/java/ro/itschool/ema/controllers/OrganizerControllerIntegrationTest.java

```diff
@@ -0,0 +1,98 @@
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.mockito.BDDMockito;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+import ro.itschool.ema.models.dtos.AddressDTO;
+import ro.itschool.ema.models.dtos.OrganizerDTO;
+import ro.itschool.ema.services.OrganizerService;
+
+import java.util.List;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.*;
+
+@SpringBootTest
+@AutoConfigureMockMvc
+class OrganizerControllerIntegrationTest {
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @MockBean
+    private OrganizerService organizerService;
+
+    @Test
+    @DisplayName("When adding a new organizer, return status code 201.")
+    void whenCreateOrganizerTest() throws Exception {
+        String organizerJson = """
+                {
+                            "organizerName": "Organizer Name",
+                            "description": "type",
+                            "website": "https://www.organizer.com/",
+                            "phoneNumber": "0712345678",
+                            "address":{
+                                "street": "Street",
+                                "city": "City",
+                                "postalCode": "123",
+                                "country": "Country"
+                            }
+                }
+                """;
+        OrganizerDTO mockOrganizerDTO = new OrganizerDTO();
+        mockOrganizerDTO.setOrganizerName("Organizer Name");
+        mockOrganizerDTO.setWebsite("https://www.organizer.com/");
+        mockOrganizerDTO.setPhoneNumber("0712345678");
+
+        BDDMockito.given(organizerService.createOrganizer(any(OrganizerDTO.class))).willReturn(mockOrganizerDTO);
+        mockMvc.perform(post("/api/organizers")
+                    .contentType(MediaType.APPLICATION_JSON)
+                    .content(organizerJson))
+                .andExpect(status().isCreated())
+                .andExpect(content().contentType(MediaType.APPLICATION_JSON))
+                .andExpect(jsonPath("$.website").value("https://www.organizer.com/")).andReturn();
+    }
+
+    @Test
+    @DisplayName("Get organizer list and return status code 200.")
+    void whenGetOrganizerList() throws Exception {
+        AddressDTO addressDTO = new AddressDTO();
+        addressDTO.setId(1L);
+        addressDTO.setStreet("Main Street 123");
+        addressDTO.setCity("City");
+        addressDTO.setPostalCode(011111);
+        addressDTO.setCountry("Country");
```

Comment:
I see an inconsistency in the way you create tests. In some you instantiate the objects inside the Tests and in others outside. Maybe you should stick to one way of doing it. Also, this instantiation could be done in a @BeforeEach method (not necessarily, of course.)